File: Anul II/Medii de proiectare si programare/Laborator/lab06/networking/src/main/java/com/ubb/protocol/response/RacesWithParticipantsResponse.java
package com.ubb.protocol.response;

import com.ubb.model.data.RaceDTO;

import java.util.Collection;

public class RacesWithParticipantsResponse implements Response {
    private final Collection<RaceDTO> races;

    public RacesWithParticipantsResponse(Collection<RaceDTO> races) {
        this.races = races;
    }

    public Collection<RaceDTO> getRaces() {
        return races;
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab07/Ubb.BikeContest.Networking/Response/RacesWithParticipantsResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ubb.BikeContest.Model.data;

namespace Ubb.BikeContest.Networking.Response
{
    [Serializable]
    public class RacesWithParticipantsResponse : IResponse
    {
        private readonly List<RaceDto> races;

        public RacesWithParticipantsResponse(List<RaceDto> races)
        {
            this.races = races;
        }

        public List<RaceDto> Races { get { return races; } }
    }
}
